server/xpub/entities/manuscript/resolvers.js

```diff
@@ -94,27 +83,28 @@ const resolvers = {
         throw new Error(errorManuscript)
       }
 
-      const manuscript = Manuscript.applyInput(
+      const manuscript = ManuscriptManager.applyInput(
         originalManuscript,
         manuscriptInput,
       )
 
-      manuscript.status = Manuscript.statuses.MECA_EXPORT_PENDING
-      await Manuscript.save(manuscript)
+      manuscript.status = ManuscriptManager.statuses.MECA_EXPORT_PENDING
+      await ManuscriptManager.save(manuscript)
 
-      mecaExport(manuscript.id, user, ip)
+      const content = 'dummy content'
```

Comment:
Is this going to be changed?

Reply:
Whoa now! Yes, that needs to be changed.